src/pages/SignupHandler/index.js

```diff
@@ -0,0 +1,267 @@
+import React from 'react';
+import {
+  Form,
+  FormGroup,
+  TextInput,
+  ActionGroup,
+  Button,
+  Split,
+  SplitItem,
+  Title,
+} from '@patternfly/react-core';
+import { CheckIcon, CloseIcon } from '@patternfly/react-icons';
+import AuthLayout from '@/components/AuthLayout';
+import { routerRedux } from 'dva/router';
+import { connect } from 'dva';
+import styles from './index.less';
+
+@connect(user => ({
+  user: user.user,
+}))
+class SignupHandler extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      password: '',
+      confPassword: '',
+      email: '',
+      variantVal: 'tertiary',
+      btnColor: 'black',
+    };
+  }
+
+  componentDidMount = () => {
+    this.disableSubmitBtn();
```

Comment:
What if we provided visual feedback to the input forms instead of disabling the button upon component mount when the user has not filled out the sign up form? We should also validate the user has entered the correct field (for example, verify user has entered an email address in the email address field). Keep in mind, this validation may be moved to the server in the future.

Reply:
Yes, we can do that as well. We should discuss these design constraints with Mary. But as per our last discussion, these constraints are not our top priority right now as it might change in future so this will need further discussion.

Reply:
We can expect this sign up form design to be constant for the foreseeable future. At the very least, we might add or remove a field; however, our component markup should be dynamic enough that this wouldn't alter core logic.

Reply:
Added visual feedback for email and password field